router/handler_ctrl/bind.go

```diff
@@ -37,9 +38,10 @@ type bindHandler struct {
 	ctrlAddressChanger CtrlAddressChanger
 	xgDialerPool       goroutines.Pool
 	linkDialerPool     goroutines.Pool
+	ctrlAddressUpdater CtrlAddressUpdater
 }
 
```

Comment:
I'd like to consolidate the address changer and updated functionality, but let's do that in a follow up PR